src/computation/Computation.h

```diff
@@ -0,0 +1,80 @@
+/*!
+ * \file Computation.h
+ *
+ * \brief Declarations for the Computation and StmtInfo structs.
+ *
+ * The Computation struct is the SPF representation of a logical computation.
+ * It contains a StmtInfo struct for each statement, which in turn contains
+ * information about that statement as mathematical objects.
+ * Originally part of spf-ie.
+ *
+ * \date Started: 10/09/20
+ *
+ * \authors Anna Rift
+ */
```

Comment:
For now it is fine to put the copyright date as University of Arizona and Boise State University 2020.